File: python/pe001.py
#!/usr/bin/env python3

""" this computes projecteuler.net problem 001 """

import sys

IMAX = 1000
total = 0

for i in range(IMAX):
    if (i % 3) == 0 or (i % 5) == 0:
        total += i

print(total)
sys.exit(0)

File: lua/pe001.lua
-- this computes projecteuler.net problem 001 
local imax = 1000
local total = 0

for i = 1, imax - 1 do
  if i % 3 == 0 or i % 5 == 0 then
    total = total + i 
  end
end

print(total)
